js/utils.js

```diff
@@ -0,0 +1,15 @@
+const getRandomPositiveInteger = (a, b) =>{
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+const createIdGenerator = () => {
+  let lastGeneratedId = 0;
+  return function (){
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+};
+const getCommentId = createIdGenerator();
```

Comment:
Создание комментария -- это уже не абстрактная функция, а предметная, т.е. это прям не какой-то там объект, не какое-то там случайное число, не какой-то там случайный элемент какого-то списка, а конкретно -- это функция получения следующего идентификатора (id) комментария, её тут быть не должно, нужно перенести в `createObject(mocks)`, т.е.  к тому модулю, который работает с этой предметной областью, а именно создает эти самые комментарии (В данном случае у нас один модуль который отвечает за создание фотографий и комментариев)
```suggestion
const getCommentId = createIdGenerator();
```

Reply:
+